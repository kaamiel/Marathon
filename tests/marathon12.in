addMovie 0 40
addMovie 0 39
addMovie 0 41
marathon 0 0
marathon 0 1
marathon 0 2
marathon 0 3
marathon 0 4
