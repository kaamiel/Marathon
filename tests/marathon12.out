OK
OK
OK
NONE
41
41 40
41 40 39
41 40 39
